### DISCUSSION TITLE
Custom SSL Certificates

### DISCUSSION THREAD

**Discussion by joaopinto14** [nextcloud/all-in-one#918] [Questions] [answered]
I've already seen that an ssl certificate is being generated for nextcloud.
I have custom certificates for my domains and I wanted to link to nextcloud.
I'm using an nginx server to do the reverse proxy
Someone can help me?

**Comment by szaimen** [comment, marked as answer]
Hi, see https://github.com/nextcloud/all-in-one/blob/main/reverse-proxy.md and https://github.com/nextcloud/all-in-one/discussions/588#discussioncomment-2811152
